Makefile

```diff
@@ -2,22 +2,23 @@ CHARM_NAME = archive-auth-mirror
 CHARM_SERIES = xenial
 CHARM_OUTPUT = build/charm-output
 RENDERED_CHARM_DIR = $(CHARM_OUTPUT)/$(CHARM_SERIES)/$(CHARM_NAME)
-RENDERED_CHARM_URI = cs:~landscape/$(CHARM_NAME)
+CHARM_URI = cs:~landscape/$(CHARM_NAME)
 
 
 .PHONY: help
 help: ## Print help about available targets
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: charm-build
+charm-build: REV_HASH = $(shell git rev-parse HEAD)
+
 charm-build:  ## Build the charm
```

Comment:
Maybe you should kill the blank line between the two 'charm-build'? I think makes it easier to see that they are connected.